UI/window-basic-main.cpp

```diff
@@ -167,6 +167,117 @@ static int CountVideoSources()
 	return count;
 }
 
+static void MIDICallback(double deltatime, std::vector<uint8_t> *message,
```

Comment:
I think we had this conversation in another thread, but "Midi" or "midi" is fine in function names.  In comments or user-facing or logged text, it should be "MIDI".